packages/daemon/bootstrap.js

```diff
@@ -81,17 +91,21 @@ function createPidfileSync(pathname) {
     });
     // emit `server` event in app
     app.emit('server', server);
-
+  
     // server listen
     await new Promise(resolve => {
-      server.listen(PORT, resolve);
+      if (HOST) {
+        server.listen(PORT, HOST, resolve);
+      } else {
+        server.listen(PORT, resolve);
+      }
     });
   } catch (err) {
     return exitProcessWithError(err);
   }
 
   process.title = 'pipcook.daemon';
-  console.info('Server is listening at http://localhost:%s, cost %ss', PORT, process.uptime());
+  console.info('Server is listening at http://%s:%s, cost %ss', HOST || '0.0.0.0', PORT, process.uptime());
```

Comment:
0.0.0.0 won't be accessible, it should be 127.0.0.1 :(

Reply:
`0.0.0.0` is the default value if the host not set, see https://nodejs.org/api/net.html#net_server_listen_port_host_backlog_callback

Reply:
But 0.0.0.0 is not working with browsers like chrome.

Reply:
But 127.0.0.1 is not correct, we bind all the network actually.

Reply:
127.0.0.1 < 0.0.0.0, and I think we don't need to make sure this address is correct exactly, otherwise, we should list all available local network address instead of 0.0.0.0, which is invalid to access.

Reply:
If this, how could we distinguish between '0.0.0.0' and '127.0.0.1', it's confusing.